source/core/owt_base/MediaFramePipeline.h

```diff
@@ -234,6 +235,10 @@ struct FeedbackMsg {
             char     buf[128];
         } rtcp;
     } data;
+    struct MsgBuffer{
+        uint32_t len;
+        char     data[128];
```

Comment:
I wonder if you should use unit8_t as the buffer type. Also in InternalTransport impl you're also using char array instead of uint8_t.